### PR TITLE
Add MeadowlarkLC device adapter

### PR DESCRIPTION
**Pull request by ieivanov**
Fixes mehta-lab/mmCoreAndDevices#2, mehta-lab/mmCoreAndDevices#3, mehta-lab/mmCoreAndDevices#4, mehta-lab/mmCoreAndDevices#5, mehta-lab/mmCoreAndDevices#6,

### CODE REVIEW

**Comment by ieivanov** [comment]
@nicost we are ready to push the MeadowlarkLC device adapter. Before that, could you please help me upload the USB driver libraries to the `3rdparty` SVN repo and create a page on https://micro-manager.org/Device_Support? Thanks!

**Comment by nicost** [comment]
Hi @ieivanov.  Can you send me the USB driver library files (or send me a private link)?  I can then add them to the 3rdparty svn repo.  I believe that you can edit the page on micro-manager.org yourself directly with the edit page link in the top (or clone https://github.com/micro-manager/micro-manager.github.io and create a PR).

**Comment by ieivanov** [comment]
Just emailed them to you, thanks. I also found the instructions for updating the wiki, I'll work on that next.

File: DeviceAdapters/MeadowlarkLC/MeadowlarkLC.cpp [hunk 1]
```diff
@@ -0,0 +1,2464 @@
...
+//
+//                IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+//                CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+//                INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES.
+
+
+#ifdef WIN32
+//   #include <windows.h>
+#define snprintf _snprintf
+#endif
```
Comment (marktsuchida) [inline DeviceAdapters/MeadowlarkLC/MeadowlarkLC.cpp:29]: This is no longer needed with modern VC++ and should be deleted.

**Review by marktsuchida — COMMENTED** [review]
Looks good to me!

It would be best if you could also add the `MeadowlarkLC.vcxproj.filters` file if you have it. That file should be tracked by Git.

Also `3rdparty-revision` in the main micro-manager repo needs to be bumped before merging this (I will do).

**Comment by ieivanov** [comment]
Thanks @marktsuchida! I've removed that bit of old code. I don't have a `MeadowlarkLC.vcxproj.filters` file. I think this device adapter should be ready to merge.

**Comment by ieivanov** [comment]
P.S. I also added a `build.xml` to pull the `mmgr_dal_MeadowlarkLC.csv` file into the MM install directory

**Comment by ieivanov** [comment]
Hey @marktsuchida, we see that the `mmgr_dal_MeadowlarkLC.csv` is not copied in the MM install directory during the build. Could you please take a quick look at my build.xml file and see if you notice any issues?

**Comment by marktsuchida** [comment]
@ieivanov The build.xml is correct (I had tested) but I forgot that not all files get added to the installer automatically. It should work now but let me know if not.

**Comment by talonchandler** [comment]
Thanks @marktsuchida and @ieivanov. The `.csv` is now being distributed with the nightly build and it's working well with the device.

**Pull request merged**